react/Text/Text.less

```diff
@@ -59,3 +59,9 @@
 .raw {
   padding-bottom: 0;
 }
+
+.bullet {
+  display: list-item;
+  list-style-type: disc;
+  margin-left: 1.5 * @gutter-width;
```

Comment:
I dont like what im about to suggest, but, if we pull this out to a mixin we can use it in places where we cannot use the style guide components themselves, not dissimilar to the text mixins. Thoughts?

Reply:
YAGNI (IMO) 😄

Reply:
We already do, i'll take it offline.

Reply:
I'd rather avoid supporting consumers using mixins without an known consumer.
We could pull this out to a mixin if and when it is required.

Thats just a preference, if you want to support mixin consumers from day one I will.

Reply:
There's already a need for it, it's not hypothetical. Check slack 😄